### ISSUE TITLE
QuickFilter LookupEditor picks the wrong column

### ISSUE THREAD

**Issue by SimeonDD** [volkanceylan/Serenity#1078]
In a nutshell: The  QickFilter works fine as a text field but when I try to link it to a selection of distinct values from the same column for some reason the LookupEditor picks the column that is in the "nameProperty" in the TS xxxxxRow.TS in ServerTypings.
Any idea what am I missing?
I suspect that my lookup class is wrong (red "?").

For convenience I put together screenshots:
![filterpickswrongcol_screen](https://cloud.githubusercontent.com/assets/19690830/17789665/334dc9a2-6561-11e6-875a-edbe97da09c1.png)
![filterpickswrongcol_code](https://cloud.githubusercontent.com/assets/19690830/17790482/a5163756-6564-11e6-952b-606254cc8291.png)

**Comment by biggikalli** [comment]
Hi @SimeonDD ,

You should not have to worry about what's in the Common>Imports folder unless you are deleting or removing a module like Northwind....at least in my experience. It should be enough to trust the "Transform All T4 Templates" function to handle everything under this "Imports" folder.

Are you looking for a cascading filter (a filter which shown values depends on chosen value in another filter)?? Are you, like we say in BI, drilling down to a specific value with filters??

There are lots of tutorials and solved issues in the forum regarding that and I have solved all the issues in lookups and filters I have come across in my Serenity projects.

Don't hesitate to shoot at me if you need code samples.

**Comment by biggikalli** [comment]
This is your solution, ditch the Quickfilter parameter and do as the red arrow in the picture:
![image](https://cloud.githubusercontent.com/assets/15879719/17827095/474ef398-6667-11e6-85a3-2c1eda037ae9.png)

**Comment by biggikalli** [comment]
Ditch the Quickfilter parameter and put the [LookupEdit.....] in the xxxRow.cs
![image](https://cloud.githubusercontent.com/assets/15879719/17827115/911175dc-6667-11e6-822c-510792fd5d22.png)

Then in your column, call that filter by it's simple name.

**Comment by SimeonDD** [comment]
Did not work. But that's OK. I am going to close this as it will take way more work than I initially expected.
Some details below only as FYI:
If I do not have "QuickFilter" declared in either the Row.cs of the Columns.cs then no filter shows at all.
Even if I have the LookupEditor moved to the Row.cs class.
I mean here the additional row of column-specific filters that shows above the grid.

If I leave "QuickFilter" I get a text field filter to show and this filter works fine.
The goal was to show a drop-down instead of a text field.
So If I add the LookupEditor the combo shows but is filled in with the non-distinct values of the Company column. 
So, for some reason that column is picked up automatically.
Looking at the code I suspect only BudgetYearLookup.cs.
In there the line is:
IdField = TextField = DefBudgetRow.Fields.FinancialYear.**PropertyName**;

And "PropertyName" happens to be what Serene picked up automatically, which happens to be the Company name column that shows inside the combo of the filter.

In short ... let's drop this as solving this could be needing deeper knowledge of the Serene framework.
